operator/pkg/certs/certs.go

```diff
@@ -212,25 +212,25 @@ func KarmadaCertEtcdClient() *CertConfig {
 // KarmadaCert is karmada certificate, it includes certificate basic message.
 // we can directly get the byte array of certificate key and cert from the object.
 type KarmadaCert struct {
-	pairName string
-	caName   string
-	cert     []byte
-	key      []byte
+	PairName string
```

Comment:
Can we avoid exposing internal fields by adding a new function to generate `KarmadaCert`, thereby minimizing the scope of changes?

Reply:
@zhzhuang-zju So, what's your suggestion?
Do you mean to introduce some `set` methods?

Reply:
How about 
```bash
func NewKarmadaCert(pairName,caName string , cert, key []byte) *KarmadaCert{
	return &KarmadaCert{
		pairName: pairName,
		caName: caName,
		cert: cert,
		key: key,
	}
}
```

Reply:
@zhzhuang-zju , thanks for the suggestion.